src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo" class="logo-image">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <a
+            href="#Apple"
+            class="nav__link is-active"
+          >Apple
+          </a>
+          <a
+            href="#Samsung" class="nav__link"
+          >Samsung
+          </a>
```

Comment:
check code styling [recommendations](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
          <a href="#" class="nav__link">
            Samsung
          </a>
```

Reply:
isn't fixed